src/LookingGlass.jl

```diff
@@ -235,5 +238,85 @@ module_recursive_globals_names(m::Module; constness=:all, mutability=:all) =
             if !isempty(names)
         ))
 
+"""
+    typetree(t::Type)
+
+Prints the type hierarchy rooted at the specified type. Note that this is intended to be used interactively, so the current implementation works by printing to `stdout`, and returns `nothing`.
+
+```julia-repl
+julia> typetree(Integer)
+Integer
+    Bool
+    Signed
+        BigInt
+        Int128
+        Int16
+        Int32
+        Int64
+        Int8
+    Unsigned
+        UInt128
+        UInt16
+        UInt32
+        UInt64
+        UInt8
+```
+"""
+function typetree(t::Type, depth=0)
+    println(repeat("    ", depth), t)
+
+    for s in subtypes(t)
+        typetree(s, depth + 1)
+    end
+
+    return nothing
+end
+
+
+
+"""
+    supertypes(x) :: Vector{Type}
+
+`supertypes(x::Type)` returns a `Vector{Type}` of all supertypes of `x`.
+
+If `x` is not a `Type`, `supertypes(x)` returns a `Vector{Type}` of `T` such that `x :: T`. 
+
+```julia-repl
+julia> supertypes(2)
+6-element Array{Type,1}:
+ Int64
+ Signed
+ Integer
+ Real
+ Number
+ Any
+
+julia> supertypes(Int64)
+5-element Array{Type,1}:
+ Signed
+ Integer
+ Real
+ Number
+ Any
+```
+"""
+function supertypes end
+
+function supertypes(t::Type, acc=[])
```

Comment:
```suggestion
function supertypes(t::Type)
```
:) It looks like this field is no longer used

Reply:
Good catch!